src/styles/globals.css

```
@@ -170,40 +171,40 @@
 @layer components {
   .footer {
     --footer-text-size: clamp(3rem, 14rem, 14vw);
+    position: relative;
```

Stylelint (17.5.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)